resources/Procfile

```diff
@@ -0,0 +1,3 @@
+//web: NEW_RELIC_CONFIG_FILE=newrelic.ini newrelic-admin run-program YOUR_COMMAND_OPTIONS
+//example
+web: NEW_RELIC_CONFIG_FILE=newrelic.ini newrelic-admin run-program gunicorn server:app
```

Comment:
Same, this would typically be something the user specifies. Is there a reason you wanted to include this?

Reply:
This was to provide examples to users, however, this is also defined in New Relic documentation, so can be removed